WORKSPACE

```diff
@@ -24,3 +24,11 @@ http_archive(
     strip_prefix = "FCGI-0.79",
     url = "https://cpan.metacpan.org/authors/id/E/ET/ETHER/FCGI-0.79.tar.gz",
 )
+
```

Comment:
Please add a comment here about this being imported as a Perl script with no extension for testing this scenario.

Reply:
Done, also mentioned that it could be useful for bazel coverage report postprocessing.